templates/devin/scripts/smoke.sh

```
@@ -0,0 +1,167 @@
+
+# ---------------------------------------------------------------------------
+# 9. wait for events, stop the stream, print what we captured
+# ---------------------------------------------------------------------------
+step "9. captured SSE events"
+sleep 30
+kill "$sse_pid" 2>/dev/null || true
+wait "$sse_pid" 2>/dev/null || true
+if [ -s "$sse_tmp" ]; then
+  cat "$sse_tmp"
+else
+  echo "(no SSE events captured — is the host authenticated with `devin auth login`?)"
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)